libnymea/interfaces/cosensor.json

```diff
@@ -6,6 +6,8 @@
             "name": "co",
             "type": "double",
             "unit": "PartsPerMillion",
+            "minValue": 0,
+            "maxValue": 50,
```

Comment:
Are you sure there is a max? 50 ppm seems not that much CO

Reply:
everything > 35 is considered severly hazardeous. Anyhow, 50 is probably a bit tight indeed. I've changed it to 100 so we can also measure straight deadly doses :D